torch/_C/_autograd.pyi

```diff
@@ -35,6 +35,14 @@ class DeviceType(Enum):
     Metal = ...
     ...
 
+class _ExperimentalConfig:
```

Comment:
These are no needed anymore right?

Reply:
actually they are because in the same file we have ProfilerConfig and this is an argument to the config 
```
        with_modules: bool,
        experimental_config: _ExperimentalConfig,  <<<<
```